cmd/runAsNonRoot.go

```diff
@@ -29,32 +30,74 @@ func checkRunAsNonRoot(container ContainerV1, result *Result) {
 	} else if container.SecurityContext == nil || container.SecurityContext.RunAsNonRoot == nil {
 		occ := Occurrence{
 			container: container.Name,
-			id:        ErrorRunAsNonRootNil,
+			id:        ErrorRunAsNonRootPSCNilCSCNil,
 			kind:      Error,
-			message:   "RunAsNonRoot is not set, which results in root user being allowed!",
+			message:   "RunAsNonRoot is not set in Container/Pod's Security Context, which results in root user being allowed!",
```

Comment:
I am not 100% sure we want to talk about the pod in the error message here. But it's not something that will hold me back from merging. It just creates weird messages when there are more than 1 container.

Reply:
Totally agree this could get confusing with multiple containers, removed Pod from the error message